docs/CONTRIBUTING.md

```diff
@@ -23,7 +29,31 @@ You'll need to generate mocks if an existing endpoint method is modified or a ne
 mockgen -source=example_resource.go -destination=mocks/example_resource_mocks.go -package=mocks
 ```
 
-## Adding a New Endpoint
+## Adding API changes that are still behind a feature flag
+
+On top of your code changes, or anywhere visible, add a comment that reads like this: 
+
+```
+// **Note: This field is still in BETA and subject to change.**
+ExampleNewField *bool `jsonapi:"attr,example-new-field,omitempty"`
+```
+
+When adding test cases, use the skipIfBeta() test helper to omit beta features from running in CI.
+
+```
+t.Run("with nested changes trigger", func (t *testing.T) {
+  skipIfBeta(t)
+  options := WorkspaceCreateOptions {
+     // rest of required fields here
+     ExampleNewField: Bool(true),
+   }
+  // the rest of your test logic here
+})
+```
+
+Once your PR has been merged you can later come back and remove the skipIfBeta() flag.
```

Comment:
I would prefix this with `**Note**:` to make it more prominent.

Reply:
Actually once the PR is merged, we don't want to remove `skipIfBeta()` until the feature either reaches GA or the flag is enabled in CI